.cargo/config.toml

```diff
@@ -1,6 +1,9 @@
+[build]
+rustflags = ["--cfg", "tokio_unstable"]
+
 [target.x86_64-unknown-linux-gnu]
 linker = "clang"
-rustflags = ["-C", "link-arg=-fuse-ld=lld"]
+rustflags = ["-C", "link-arg=-fuse-ld=lld", "--cfg", "tokio_unstable"]
```

Comment:
Will this be baked into release builds? We should probably avoid having unstable features in releases. Right now Fedimint is unstable itself, but I fear that we will forget about it.

Reply:
AFAIU "unstable" here means, that *compilation* could break in minor releases, not that it's dangerous at runtime: https://github.com/tokio-rs/tokio/blob/1dcfe1cc9bb32fdc57ead3ff642d58bc8f282fd4/tokio/Cargo.toml#L114

Having it in prod builds is where I actually want to check it out. :)